tests/MyGame/Example/Ability.cs

```diff
@@ -47,7 +47,13 @@ public struct Ability : IFlatbufferObject
 
 public class AbilityT
 {
+#if ENABLE_JSON_SERIALIZATION
+  [Newtonsoft.Json.JsonProperty("id")]
+#endif
   public uint Id { get; set; }
+#if ENABLE_JSON_SERIALIZATION
```

Comment:
I am wondering if it makes sense to make this a flag to `flatc` rather than a preprocessor flag, since this is generated code, it would make more sense to simply omit this code completely if the flag is not specified

Reply:
because C# generated code contains `ENABLE_SPAN_T` so I chose the preprocessor option.
It's easy, I will try to rewrite it.

Reply:
add `cs-gen-json-serializer` flag. and add this flag to `generate_code.bat`.